src/cmd/services/m3coordinator/ingest/write.go

```diff
@@ -365,9 +374,12 @@ func (d *downsamplerAndWriter) writeToStorage(
 		multiErr xerrors.MultiError
 		errLock  sync.Mutex
 	)
-
-	for _, p := range storagePolicies {
+	if len(storagePolicies) > 1 {
+		d.metrics.multiPolicyWritten.reportInc(source, len(storagePolicies))
```

Comment:
do we still need this?

Reply:
Prefer to keep it around sot that if anything changes later.